webui/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "webui",
+  "version": "0.0.0",
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "build:watch": "vite build --watch",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "apexcharts": "^3.45.1",
+    "crypto-js": "^4.2.0",
+    "vue": "^3.3.11",
+    "vue3-apexcharts": "^1.4.4"
+  },
+  "devDependencies": {
+    "@vitejs/plugin-vue": "^4.5.2",
+    "autoprefixer": "^10.4.16",
+    "postcss": "^8.4.33",
+    "tailwindcss": "^3.4.1",
```

Comment:
We can switch from tailwind to unocss. It works great with Vue and Vite, has more functionality and higher performance, it is also easy to configure and is fully compatible with tailwind, so you don’t have to rewrite anything.

We can replace Autoprefixer and postcss with lightningcss.